src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs

```diff
@@ -24,7 +24,7 @@ public Http3LoopbackServer(QuicImplementationProvider quicImplementationProvider
         {
             options ??= new GenericLoopbackOptions();
 
-            _cert = Configuration.Certificates.GetSelfSigned13ServerCertificate();
+            _cert = Configuration.Certificates.GetServerCertificate();
```

Comment:
@wfurt I don't know if this change is what we want/need. but it did help with some of the failing tests.

Reply:
I don't think it really matters. CI may be set up to trust your contoso chain but not self-signed. Either one should be fine once we have custom validation in place.